app/templates/search.hbs

```diff
@@ -1,2 +1,40 @@
-Search Page
-{{outlet}}
+
+<div id="bg">
+  <img src="https://s3-us-west-1.amazonaws.com/waldojobs-dev/app-assets/images/source/welcomeBG3.jpg"/>
+</div>
+<div class="wrapper">
+<img class="search-page-logo" src="https://s3-us-west-1.amazonaws.com/waldojobs-dev/app-assets/images/source/waldo_logoColor.png"/>
```

Comment:
BTW, I made a helper for icons and images. This image could be defined by 
`src={{image-asset "waldo_logoColor.png"}}`

the background could be `src={{icon-asset "welcomeBG3.jpg"}}`

just so ya know :)

Reply:
Nice,  will use this.